src/sidebar/components/NotebookView.js

```diff
@@ -21,37 +23,88 @@ import ThreadList from './ThreadList';
  */
 function NotebookView({ loadAnnotationsService }) {
   const store = useStoreProxy();
+
+  const filters = store.getFilterValues();
```

Comment:
`NotebookView` has been made into a "clearing house" of store-state-related management, versus having this distributed throughout its children.